src/Commands/ConvertFont.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace Barryvdh\DomPDF\Commands;
+
+use Dompdf\Dompdf;
+use Exception;
+use FontLib\Font;
+use Illuminate\Console\Command;
+
+class ConvertFont extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'font:convert
+    {fontFamily : the name of the fonts, e.g. Verdana, \'Times New Roman\', monospace, sans-serif. If it equals to "system_fonts", all the system fonts will be installed.}
+    {fileName : the .ttf or .otf file for the normal, non-bold, non-italic face of the fonts.}
+    {--b|bold= : Bold fonts face specific file}
+    {--i|italic= : Italic fonts face specific file}
+    {--bi|bold-italic=} : Bold italic fonts face specific file';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'If the optional b|i|bi files are not specified, this command will search
+the directory containing normal fonts file (file-name) for additional files that
+it thinks might be the correct ones (e.g. that end in -Bold, _Bold or b or B).  If
+it finds the files they will also be processed.  All files will be
+automatically copied to the DOMPDF fonts directory, and afm files will be
+generated using php-fonts-lib (https://github.com/PhenX/php-fonts-lib).';
+
+    const SUPPORTED_FILES = ['.ttf', '.otf'];
+
+    public static $PATTERN_REGULAR = ['-Regular', '_Regular'];
+    public static $PATTERN_BOLD = ['_Bold', '-Bold', 'b', 'B', 'bd', 'BD'];
+    public static $PATTERN_ITALIC = ['_Italic', '-Italic', 'i', 'I'];
+    public static $PATTERN_BOLD_ITALIC = ['_Bold_Italic','_BoldItalic', '-Bold_Italic', '-BoldItalic', 'bi', 'BI', 'ib', 'IB'];
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $dompdf = new Dompdf();
```

Comment:
The Dompdf instance must be taken from the service container.
1. Then you won't need to manually set the option for `font_dir`. The service container itself will set all options from `config('dompdf.defines')`;
2. And in such a case, the `FontMetrics` class will be initialized with the `fontDir` option in mind. And then the fonts will not be overwritten, as described here https://github.com/barryvdh/laravel-dompdf/pull/820#issuecomment-1153099946